src/pages/components/structured-list/style.mdx

```diff
@@ -52,34 +126,40 @@ set in sentence case. All typography is left aligned.
 |                 | padding-left, padding-right | 16 / 1      | `$spacing-05` |
 | Icon            | height, width               | 16 / 1      | –             |
 
+### Default structure
+
 <div className="image--fixed">
 
-![Spacing and measurements for Structured List](images/structured-list-style-1.png)
+![Spacing and measurements for default structured list](images/structured-list-style-5.png)
 
 </div>
 
 <Caption>
-  The width of structured list varies based on content and layout.
+  Spacing and measurements for default structured list with hang and flush
+  alignment | px / rem.
 </Caption>
 
+### Selectable structure
+
 <div className="image--fixed">
 
-![Spacing and measurements for structured list with selection](images/structured-list-style-2.png)
+![Spacing and measurements for selectable structured list](images/structured-list-style-6.png)
 
 </div>
 
 <Caption>
-  Spacing and measurements for default spacing with hang alignment and flush
-  alignment| px / rem
+  Spacing and measurements for selectable structured list with hang alignment |
+  px / rem.
 </Caption>
 
+## Size
+
+There are two structured list sizes: default and condensed.
+
 <div className="image--fixed">
 
-![Spacing and measurements for structured list with selection](images/structured-list-style-3.png)
+![Sizes of structured list with selection](images/structured-list-style-7.png)
```

Comment:
This image in the Size section could be simplified a bit with the specs, to direct the users attention more to the specifications of what changes between the condensed and default  heights. I will create an example of what I mean in your figma file.